App.js

```diff
@@ -3,24 +3,23 @@ import { TailwindProvider } from "tailwind-rn";
 import utilities from "./tailwind.json";
 
 import Base from "./screens/base";
-//import Index from "./screens/index";
-//import ListView from "./screens/listView";
-//import Login from "./screens/login";
-//import SignUp from "./screens/signUp";
-//import AccountRecovery from "./screens/accountRecovery";
-import DbTest from "./screens/dbTest";
+// import Login from "./screens/login";
+import Index from "./screens/index";
+// import ListView from "./screens/listView";
+// import SignUp from "./screens/signUp";
+// import AccountRecovery from "./screens/accountRecovery";
+// import DbTest from "./screens/dbTest";
 
 export default function App() {
   return (
     <TailwindProvider utilities={utilities}>
       <Base>
         {/* <Login /> */}
-        {/* <Index /> */}
-        {/*<ListView />*/}
-        {/* <MenuList/> */}
+        <Index />
+        {/* <ListView /> */}
         {/* <SignUp/> */}
-        {/*<AccountRecovery/>*/}
-        <DbTest />
+        {/* <AccountRecovery/> */}
+        {/* <DbTest /> */}
```

Comment:
Let's get rid of this database test stuff. This was just for the in class demo last semester.